GoogleDataTransport/GDTCORLibrary/GDTCOREvent.m

```diff
@@ -14,11 +14,11 @@
  * limitations under the License.
  */
 
-#import <GoogleDataTransport/GDTCOREvent.h>
+#import "GDTCORLibrary/Public/GDTCOREvent.h"
```

Comment:
for public header you might have to use this format? 
#import <GoogleDataTransport/GDTCOREvent.h>

Reply:
You only need to use that format in other public headers, here's it's fine.